Configurations.md

```diff
@@ -1040,6 +1040,14 @@ fn add_one(x: i32) -> i32 {
 }
 ```
 
+## `format_cargo_toml`
+
+Format `Cargo.toml` files.
+
+- **Default value**: `false`
+- **Possible values**: `true`, `false`
+- **Stable**: No (tracking issue: [#4091](https://github.com/rust-lang/rustfmt/issues/4091))
+
```

Comment:
Is it possible to add an example of a minimal `Cargo.toml` file getting reformatted, kind of like other option. At the very least could we elaborate on what formatting we do here or at least link out to where the formatting rules are defined.